requirements.txt

```diff
@@ -15,7 +13,7 @@ pandas==1.1.4
 plotly==4.12.0
 pylint==2.6.0
 pytest-cov==2.10.1
-python-coveralls==2.9.3
+coveralls==2.1.2
```

Comment:
This is very nitpicky, but can you maintain the alphabetical order in `requirements.txt`

Reply:
Didn't spot the alphabetical order! Thanks, will update.